rosetta-source/pom.xml

```diff
@@ -1034,6 +1080,12 @@
             <artifactId>mockito-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <!-- org.isda.CodeListTransformer dependency -->
+        <dependency>
+            <groupId>net.sf.saxon</groupId>
+            <artifactId>Saxon-HE</artifactId>
+            <version>10.6</version> <!-- Adjust as needed -->
+        </dependency>
```

Comment:
Versions are managed centrally under `dependencyManagement` in the parent pom. Please move it there.